### ISSUE TITLE
projeto duplicado e migracao para gitlab

### ISSUE THREAD

**Issue by lucasmation** [Ipeadata/geras#3]
@eriveltonguedes este projeto parece ser uma duplicação do projeto Ipeadata-geras que foi migrado para o gitlab link [https://gitlab.com/ipeadata/ipeadata_scripts_preparar_dados](url) podemos deletar este também?

**Comment by eriveltonguedes** [comment]
sim, podem deletar tudo e deixar somente o oficial. Temos copias locais e
se algo der errado atualizaremos com as copias.

abs

Atenciosamente,

Erivelton Pires Guedes -
Instituto de Pesquisa Econômica Aplicada (Ipea) - (21) 3515-8548
http://www.ipea.gov.br

**Comment by eriveltonguedes** [comment]
Pessoal, testei em  casa mas também ficou com aparência ruim (num
computador que jamais havia comparecido ao site).

abs




Atenciosamente,

Erivelton Pires Guedes -
Instituto de Pesquisa Econômica Aplicada (Ipea) - (21) 3515-8548
http://www.ipea.gov.br


2017-08-28 22:06 GMT-03:00 Erivelton Guedes <eriveltonguedes@gmail.com>:
